solarforecastarbiter/io/reference_observations/srml.py

```diff
@@ -209,8 +209,9 @@ def initialize_site_forecasts(api, site):
     site : datamodel.Site
         The site object for which to create Forecasts.
     """
-    common.create_forecasts(api, site, srml_variable_map.values(),
-                            default_forecasts.TEMPLATE_DETERMINISTIC_FORECASTS)
+    common.create_forecasts(
+        api, site, srml_variable_map.values(),
+        default_forecasts.TEMPLATE_DETERMINISTIC_NWP_FORECASTS)
```

Comment:
SRML now has a few PV systems, so I think we should support the full reference forecast suite for the network.

Reply:
Also noting that this line and the corresponding one in `crn.py` are not tested. OK to address that separately.

Reply:
it'll add 20-30% more forecasts to make, but should be ok